### PR TITLE
BaSP-TG-21: get getbyID post tasks

### PR DESCRIPTION
**Pull request by ProDIGGY-vinilos**

### CODE REVIEW

**Comment by sofihoj** [comment]
The name of the PR should follow the format BaSP-TG-XX: Short description

File: src/tests/tasks.test.js [hunk 1]
```diff
@@ -1,5 +1,116 @@
...
+
+  test('Should return 1 task', async () => {
+    const response = await request(app).get(`/tasks/${reqId}`).send();
+    // eslint-disable-next-line no-underscore-dangle
+    expect(response.body.data._id).toContain(reqId);
+  });
+
+  test('Should return status code 404', async () => {
+    const response = await request(app).get(`/tasks/${badReqId}`).send();
+    expect(response.status).toBe(404);
```
[the remark this answers is not in the document]
Comment (ProDIGGY-vinilos) [reply src/tests/tasks.test.js:57]: that validations are right down

File: src/tests/tasks.test.js [hunk 2]
```diff
@@ -1,5 +1,116 @@
...
+
+  test('Should return status code 404', async () => {
+    const response = await request(app).get('/tasks/456456').send();
+    expect(response.status).toBe(404);
+  });
+
+  test('Should return error true', async () => {
+    const response = await request(app).get('/tasks/456456').send();
+    expect(response.status).toBeTruthy();
+  });
```
[the remark this answers is not in the document]
Comment (ProDIGGY-vinilos) [reply src/tests/tasks.test.js:78]: its another error, its validate that the id are in invalid format

**Comment by ProDIGGY-vinilos** [comment]
> 
Sorry, my bad

**Comment by SantiagoGonzalez97** [comment]
Maybe you should put the "expect(response.body.message)"

**Review by Alfondalix — APPROVED** [review]

**Pull request merged**